City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-5330: KASKO debug page error

The debug page on KASKO produced a WSOD error as it tried to fetch releases for `helfi_group` custom module. This has now been fixed with the changes done in this PR and the one in `hdbt_admin`.

## What has been done
- removed `helfi_group` custom module from `composer.json` as custom modules are always installed and enabled
- added requirements and patches from `helfi_group` module's `composer.json`
- removed `helfi_group` module's `composer.json`

## How to install
- set up KASKO instance
- get the changes from this branch: `git checkout UHF-5330_kasko_debug_error`
- get the equivalent branch of `hdbt_admin`: `composer require drupal/hdbt_admin:dev-UHF-5330_kasko_debug_error`

## How to test
- access the debug page and see that there are no errors

See the `hdbt_admin` PR here: https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/131
